k8s-test.bash

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -x
+
+CLUSTER_CLIENT="${CLUSTER_CLIENT:-$(which kubectl)}"
+
+if ! command -v "${CLUSTER_CLIENT}"; then
+    echo "${CLUSTER_CLIENT} command not found in \$PATH"
+    exit 1
+fi
+
+if ! command -v helm; then
+    echo "No 'helm' found in \$PATH"
+    exit 1
+fi
+
+TEST_NAMESPACE="${TEST_NAMESPACE:-k8s-test}"
+RELEASE_NAME="${RELEASE_NAME:-cryostat-k8s-test}"
+
+function cleanup() {
+    "${CLUSTER_CLIENT}" delete ns "${TEST_NAMESPACE}"
+    if [ "${CREATE_CLUSTER:-true}" = "true" ]; then
+        kind delete cluster
+    fi
+}
+trap cleanup EXIT
+cleanup
```

Comment:
Maybe not `cleanup` here? When I don't have a cluster running , the command `kubectl delete` will hang for quite sometimes (i.e. trying to contact an old cluster I have elsewhere that is still in kubeconfig).

Reply:
Ah, fair enough. It doesn't hang for very long for me, but I guess it depends on the existing configurations.